client/components/Book.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import Dates from './Dates.jsx';
 import Guests from './Guests.jsx';
 import PriceSummary from './PriceSummary.jsx';
-import { ButtonSubmit, CenterText } from './styled/Styled.jsx';
+import { ButtonSubmit, CenterText, BookStyle } from './styled/Styled.jsx';
```

Comment:
1) The ButtonSubmit style was used in an earlier commit so prob didnt show up in this PR diff.
2) I called it "CenterText" because I was planning to use it whenever I had to style text that was centered in the module. I sometimes would make styled components and realize that I only had to use it once, or other parts of the module actually requires slight modifications and that original component can't be reused exactly without extending another component or making a new one.
3) I called it "BookStyle" to differentiate it from my Book component. I maybe should've called it "Booking" component but I didn't want to confuse that with the "Booking Module".